test_haystack/solr_tests/server/wait-for-solr

```diff
@@ -8,7 +8,8 @@ import requests
 max_retries = 100
 retry_count = 0
 retry_delay = 15
-status_url = "http://localhost:9001/solr/collection1/admin/ping"
+# status_url = "http://localhost:9001/solr/collection1/admin/ping"
```

Comment:
This will break anyone working locally. We should probably make this something like this:

```python
solr_url = os.environ.get("SOLR_URL", "http://localhost:9001/solr/collection1/")
status_url = f"{solr_url}admin/ping"
```

(we could do other things with the hostname but I figure this is easies to make it possible to customize the protocol, hostname, port, or collection name)

Reply:
I would not paint us into a corner.  I was just trying to see if your suggestion worked.  Unfortunately, it does not.
```
Unhandled exception requesting http://solr:9001/solr/collection1/admin/ping: HTTPConnectionPool(host='solr', port=9001): Max retries exceeded with url: /solr/collection1/admin/ping (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbde5626a70>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Waiting 15 seconds for Solr to start (retry #82, status 0)
```

Reply:
I'm wondering how this originally worked because Solr's standard port is 8983 and we don't appear to be customizing that anywhere. I suspect that changing the port number on https://github.com/django-haystack/django-haystack/blob/ffcd9bf8f20ca410e42d950114a99e4c557e726e/.github/workflows/test.yml#L50 will fix that but that should have broken before now.